src/Observables.jl

```diff
@@ -1,10 +1,10 @@
-__precompile__()
-
 module Observables
 
 export Observable, on, off, onany, connect!, obsid, async_latest, throttle
+export name, name!, rename!
```

Comment:
These seem like overly generic names to export. (Then again, `on` and `off` are exported by this package. Yikes!)